Blackjack.Tests/ControllerShould.cs

```diff
@@ -20,7 +20,7 @@ public ControllerShould()
             _mockDeck = new Mock<IDeck>();
             var player = new Player(new Hand());
             var dealer = new Dealer(new Hand());
-            var players = new List<IPlayer> { player, dealer };
+            var players = new List<IParticipant> { player, dealer };

```

Comment:
matching variable name? if you're wanting to avoid confusion as much as possible